pkg/webhook/pod/tikv_deleter.go

```diff
@@ -144,6 +145,14 @@ func (pc *PodAdmissionControl) admitDeleteUselessTiKVPod(payload *admitPayload)
 		}
 	}
 
+	isUpgrading := operatorUtils.IsStatefulSetUpgrading(payload.ownerStatefulSet)
+	if isUpgrading {
+		// If the tikv is both during upgrading and scaling-in, we would take it as upgrading
```

Comment:
Is it OK to generate two events for this case?

Reply:
I suggest not as it is misleading.

Reply:
Normal upgrading should not be in this function, right?
There are other functions that upgrading may be hanled in, such as `admitDeleteUpTiKVPod`, etc.

Reply:
updated.